dashboards/jenkins/JenkinsDashboard.json

```diff
@@ -0,0 +1,831 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "id": 59,
```

Comment:
this needs to be `null` - when you export the dashboard json from grafana, make sure to checkmark the "Export for sharing externally" field!

![image](https://user-images.githubusercontent.com/5897740/63297853-2fcf5080-c287-11e9-987c-2d965bb87997.png)